docs/index.rst

```diff
@@ -0,0 +1,24 @@
+.. bioconda-recipes documentation master file, created by
```

Comment:
So, if we modify this to contain the contents of our current homepage, and install the generated HTML to the root of bioconda.github.io instead of packages, we effectively replace our homepage with a nice integrated Sphinx version, right? I think we should do that, but we can do it in a secondary PR.